### DISCUSSION TITLE
Laravel Shopper

### DISCUSSION THREAD

**Discussion by mckenziearts** [HassanZahirnia/laravel-package-ocean#93] [Package Suggestions]
Package to add e-commerce admin in your laravel project

Repo: https://github.com/shopperlabs/framework

Categorie: E-commerce

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!

Although it appears your package does not support Laravel 10. I encourage you to add support for that if that's the case. As we'll be removing any packages that are not compatible with the latest active Laravel version, every once in a while.
